docs/tutorials/transaction-frames.md

```diff
@@ -0,0 +1,305 @@
+---
+sidebar_label: Frames Transactions in Open Frames
```

Comment:
How about just Transactions in Open Frames? Might be redundant especially for a sidebar